server/notification-providers/zoho-cliq.js

```diff
@@ -87,24 +82,30 @@ class ZohoCliq extends NotificationProvider {
                 return okMsg;
             }
 
-            let url;
+            let address;
+
             switch (monitorJSON["type"]) {
-                case "http":
-                case "keywork":
-                    url = monitorJSON["url"];
+                case "ping":
+                    address = monitorJSON["hostname"];
                     break;
-                case "docker":
-                    url = monitorJSON["docker_host"];
+                case "port":
+                case "dns":
+                case "gamedig":
+                case "steam":
+                    address = monitorJSON["hostname"];
```

Comment:
i know but i just wanted to avoid making the changes out of the scope of cliq notification... maybe this optimization should be moved to a different PR

Reply:
Let's do this in this PR instead

Reply:
Ok so I should:
Unify code btw these 4 notifier
Move the code to a superclass
Call this method from the 4 notifier code

For now I think I'll give up.